faunadb-scala/src/main/scala/faunadb/streaming/BodyValueFlowProcessor.scala

```diff
@@ -0,0 +1,83 @@
+package faunadb.streaming
+
+import java.nio.ByteBuffer
+import java.util
+import java.util.concurrent.{Flow, SubmissionPublisher}
+
+import com.fasterxml.jackson.core.JsonProcessingException
+import com.fasterxml.jackson.databind.{JsonMappingException, ObjectMapper}
+import faunadb.errors.StreamingException
+import faunadb.values.{StringV, VSuccess, Value}
+import org.slf4j.LoggerFactory
+
+import scala.collection.JavaConverters.asScalaIteratorConverter
+
+class BodyValueFlowProcessor extends SubmissionPublisher[Value] with Flow.Processor[util.List[ByteBuffer], Value] {
```

Comment:
I _think_ we might want to define `BodyValueFlowProcessor` as private, since it should not be used by users directly, right?

```scala
private [faunadb] class BodyValueFlowProcessor
```